h2olog

```diff
@@ -93,7 +93,11 @@ quic_bpf = """
 #define MAX_HEADER_VALUE_LEN 128
 #define TOKEN_PREVIEW_LEN 8
 
-int sprintf(char * restrict str, const char * restrict format, ...);
+#include <linux/kernel.h> /* sprintf */
```

Comment:
Including the entire kernel header feels like overkill. If you insist on keeping it, my suggestion is to remove the comment.

Reply:
I see. I think including it is necessary when we try some other functions (e.g. `snprintf`, which doesn't compile, though).

So I've just removed it (8a8c600de5dc1d9a7f4b83269fddc80ebd3dd045).